api2-war/src/main/java/eu/europeana/api2/v2/web/controller/SearchController.java

```diff
@@ -294,43 +300,63 @@ public ModelAndView searchJsonGet(
-                .setParameter("fl", IdBeanImpl.getFields(getBeanImpl(clazz)))
-                .setSpellcheckAllowed(false);
+        Query query = new Query(SearchUtils.rewriteQueryFields(SearchUtils.fixBuggySolrIndex(queryString))).setApiQuery(
+                true)
+                                                                                                           .setRefinements(
+                                                                                                                   refinementArray)
+                                                                                                           .setPageSize(
+                                                                                                                   rows)
+                                                                                                           .setStart(
+                                                                                                                   start -
+                                                                                                                   1)
+                                                                                                           .setSort(sort)
+                                                                                                           .setCurrentCursorMark(
+                                                                                                                   cursorMark)
+                                                                                                           .setParameter(
+                                                                                                                   "facet.mincount",
+                                                                                                                   "1")
+                                                                                                           .setParameter(
+                                                                                                                   "fl",
+                                                                                                                   IdBeanImpl
+                                                                                                                           .getFields(
+                                                                                                                                   getBeanImpl(
+                                                                                                                                           clazz)))
+                                                                                                           .setSpellcheckAllowed(
+                                                                                                                   false);
```

Comment:
Please change the indentation of this code so it's better readable

Reply:
done